src/test/java/de/retest/recheck/cli/subcommands/CommitIT.java

```diff
@@ -53,4 +55,17 @@ public void commit_should_not_accept_the_report_because_there_are_no_differences
 		cut.run();
 		assertThat( systemOutRule.getLog().contains( expected ) ).isTrue();
 	}
+
+	@Test
+	public void commit_should_accept_and_update_the_sut_state_file() throws Exception {
+		final File resultFile = new File( ReportCreator.createReportFileWithDiffs( temp ) );
+
+		final String expected = "Updated SUT state file";
+		final String[] args = { "--all", resultFile.getAbsolutePath() };
+		final Commit cut = new Commit();
+		final ParseResult cmd = new CommandLine( cut ).parseArgs( args );
+
+		cut.run();
+		assertThat( systemOutRule.getLog().contains( expected ) ).isTrue();
```

Comment:
Since `SystemOutRule#getLog()` returns a string, you should be able to do something like:

```java
assertThat( systemOutRule.getLog() ).contains( expected );
```

Reply:
Good catch.